config/routes.rb

```diff
@@ -21,6 +21,7 @@
   get '/search', to: 'search#items'
 
   namespace :admin do
+    put 'items/:id', to: 'items#update_status', :as => 'item_status'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.